### DISCUSSION TITLE
Would it be possible to call `EntityDeathEvent` {stacked amount} times when stacked entity is killed?

### DISCUSSION THREAD

**Discussion by AuroraLS3** [OmerBenGera/WildStacker#391] [Denied Feature Requests] [answered]
Hi,

My plugin counts the number of killed mobs, and with stacked entities only one death event is called which leads to inaccurate kill-count from the user's perspective.

If the event were called multiple times for the stacked entity this would solve the inaccuracy.

**Comment by OmerBenGera** [comment, marked as answer]
No, laggy af.
You can use StackedEntity#getStackAmount while in EntityDeathEvent to get the amount of times the mob has died.
